LICENSE

```diff
@@ -0,0 +1,201 @@
+                                Apache License
```

Comment:
To remove from this PR until license to use is agreed with original author.

Reply:
sadly, I think we will have to merge without indeed, removed.